build/commands/lib/util.js

```diff
@@ -516,7 +451,6 @@ const util = {
       util.copyRedirectCC()
       util.updateOmahaMidlFiles()
     }
-    if (process.platform === 'linux') util.prepareWidevineCdmBuild()
```

Comment:
removed in the last PR since function was deleted from https://github.com/brave/brave-core/pull/7495/files#diff-6149b8e49614376fc44e8cd42d0ce8661e754d14d3bb4df131c3de178e852b0dL396